interpreter/exec/simd.ml

```diff
@@ -315,10 +331,46 @@ struct
       let num_lanes = lanes F64x2
     end)
 
+  let clamp x low high = min (max x low) high
+
+  (* Narrow two v128 into one v128 by using to_shape on both operands,
+   * concatenating them, clamping the element to between low and high,
+   * then of_shape to reconstruct a v128. *)
+  let narrow to_shape of_shape low high x y =
+    let xy = (to_shape x) @ (to_shape y) in
+    let narrow i = Int32.(clamp i (of_int low) (of_int high)) in
```

Comment:
Nit: directly pass `low` and `high` as int32 to avoid the conversions.

Reply:
Done (I never knew about the `l` suffix, thanks!)